trl/trainer/ppo_trainer.py

```diff
@@ -875,6 +879,17 @@ def create_model_card(self, path: str, model_name: Optional[str] = "TRL Model")
         with open(os.path.join(path, "README.md"), "w", encoding="utf-8") as f:
             f.write(model_card_content)
 
+    def set_seed(self, seed: Optional[int]):
+        """
+        Helper function for reproducible behavior to set the seed in `random`, `numpy`, `torch` and/or `tf` (if installed).
+        Args:
+            seed (`int`): The seed to set.
+        """
+        random.seed(seed)
+        # np.random.seed(seed)      # if np is added to functionality, add this line
```

Comment:
lets maybe add it - don't know if torch or some other library is using numpy somewhere.